cmd/devp2p/internal/ethtest/chain.go

```diff
@@ -22,15 +22,17 @@ type Chain struct {
 	chainConfig *params.ChainConfig
 }
 
-func (c *Chain) WriteTo(writer io.Writer) (int64, error) {
+/* unused & causing go vet warning
```

Comment:
do we need to actually disabled this? as this is go-ethereum code

Reply:
also, the commented code is different from the original of geth. we shouldn't change.

Reply:
Maybe not, it's just causing go vet error and it is not used

Reply:
thinks that are not used but are from geth, we should leave. as next geth upgrade they might start using and we will receive a compile error as this code won't be there (as we removed from our history).